Index: buildtools/bin/bash_functions.sh
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>#!/bin/bash\n\nsource ~/Sety-project/devops/buildtools/bin/variables.sh\nsource ~/Sety-project/devops/buildtools/bin/git_functions.sh\nsource ~/Sety-project/devops/buildtools/bin/simex_functions.sh\nsource ~/Sety-project/devops/buildtools/bin/python_functions.sh\n\nj() {\n\tREPOS=(config mktdata .cache .ssh .aws)\n\t\n\tif [[ \" ${REPOS[*]} \" =~ \" $1 \" ]]; then\n\t\tcd ~/$1\n\t\treturn\n\tfi\n\n\tcd ~/Sety-project/$1\n}\n\njpl() {\n    cd ~/Sety-project/pylibs/$1\n}\n\nopen_logs() {\n    libreoffice /tmp/pnl/portfolio_history_ftx_SysPerp.xlsx /tmp/tradeexecutor/all_by_symbol.csv\n}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/buildtools/bin/bash_functions.sh b/buildtools/bin/bash_functions.sh
--- a/buildtools/bin/bash_functions.sh	(revision a6dd4151704cc8bd197741fe3714801effe7497f)
+++ b/buildtools/bin/bash_functions.sh	(date 1655913573080)
@@ -21,5 +21,6 @@
 }
 
 open_logs() {
+    cache_tmp
     libreoffice /tmp/pnl/portfolio_history_ftx_SysPerp.xlsx /tmp/tradeexecutor/all_by_symbol.csv
 }
\ No newline at end of file
